packages/status-js/src/request-client/request-client.ts

```diff
@@ -127,57 +143,80 @@ class RequestClient {
     return mapUser(contactCodeAdvertisement, publicKey)
   }
 
-  private fetchCommunityDescription = async (
-    /** Uncompressed */
-    publicKey: string
-  ): Promise<CommunityDescription | undefined> => {
-    const contentTopic = idToContentTopic(publicKey)
-    const symmetricKey = await generateKeyFromPassword(publicKey)
+  public fetchCommunityDescription = async (
```

Comment:
Exposing raw unlike `fetchCommunity` since I'm not sure what the dapp's mappings needs are.